docs/PLUGINS.md

```diff
@@ -27,7 +27,15 @@ Copies the `arg` of the selected item. (Uses modifiers where necessary)
 
 #### exec
 
-Executes a node script. (Uses modifiers where necessary)
+Executes a the node script `arg.script` of the selected item and is passed the array `arg.arg` as an argument.
+The script can retrieve the arguments like this:
+
+```js
+var arg = process.argv // Returns an array of arguments:
```

Comment:
Constants should be used when you don't need reassignment so we should use it whenever we can.

If you need a variable, using the `let` keyword would allow for a block scoped variable. The `var` keyword is scoped to the function block so it can cause some issues in some situations.

Both `const` and `let` was made available in the ES2015 spec the past year.